css/zocial.svg

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
 <!--
-2016-11-28: Created with FontForge (http://fontforge.org)
+2017-1-22: Created with FontForge (http://fontforge.org)
 -->
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
 <metadata>
-Created by FontForge 20161116 at Mon Nov 28 13:14:41 2016
- By 尚享樊
-Copyright (c) 2016, &#229;&#176;&#154;&#228;&#186;&#171;&#230;&#168;&#138;
+Created by FontForge 20161013 at Sun Jan 22 20:17:13 2017
+ By winnie,,,
+Copyright (c) 2017, winnie,,,
```

Comment:
It looks like fontforge currently doesn't support explicitly specifying an author. I have opened an issue about that: https://github.com/fontforge/fontforge/issues/3004